src/styles/content/tables/table-cell-inactive/styles.less

```diff
@@ -0,0 +1,131 @@
+& when (@table-cell-inactive-enabled) {
+  /* Table */
+
+  .table {
+
+    tr.inactive td,
+    td.inactive {
+      .element-text-style(table-cell-inactive);
```

Comment:
Let's not use these mixins; instead just write the actual CSS properties. It will reduce this down to just a few lines of code.